src/components/pages/login/login-form.tsx

```diff
@@ -56,22 +59,33 @@ export function LoginForm() {
             {errors.password?.message}
           </FormErrorMessage>
         </Flex>
-        <Input
-          type="password"
-          placeholder="Min. 8 characters"
-          size="lg"
-          {...register('password', { required: '비밀번호를 입력해 주세요.' })}
+        <Controller
+          name="password"
+          control={control}
+          rules={{
+            required: { value: true, message: '비밀번호를 입력해주세요.' },
+          }}
+          render={({ field: { onChange, value } }) => (
+            <PasswordInput
+              onChange={onChange}
+              value={value}
+              id="newPassword"
+              placeholder="비밀번호를 입력해주세요."
+            />
+          )}
         />
       </FormControl>
       <Flex align="center">
-        <Switch id="save-login-info" mr={3} />
+        <Switch id="save-login-info" mr={3} colorScheme="black" />
         <FormLabel htmlFor="save-login-info" mb={0}>
           로그인 정보 저장
         </FormLabel>
         <Spacer />
-        <Button variant="link">비밀번호를 잊으셨나요?</Button>
+        <Link href="/">
+          <b>비밀번호를 잊으셨나요?</b>
+        </Link>
```

Comment:
P5: 페이지 경로들은 상수로 관리하는 것이 어떨까요?

```suggestion
import { ROUTES } from '@/constants/routes';

        <Link href={ROUTES.HOME}>
          <b>비밀번호를 잊으셨나요?</b>
        </Link>
```

이런 식으로 하면 페이지 이동 로직과 관련해서 휴먼 에러를 줄일 수 있을 것 같아요!

Reply:
오 라우트까지 이렇게 상수로 관리하는 것은 처음인데 괜찮아 보이네요? 말씀주신대로 routes 파일 만들어보겠습니다~

Reply:
적용했습니다!